install/resources_generator.py

```diff
@@ -480,6 +485,7 @@ def _set_containers(self, role):
             nodes = self.drive_nodes[:]
         else:
             nodes = self.compute_nodes[:]
+        failure_domain = "" if self.args.use_auto_failure_domain else get_failure_domain_based_on_nodename()
```

Comment:
`failure_domain` is not used, i guess it should be passed in line 493 in the `Container` instantiation

Reply:
you are totally right